tests/integration/clientside/assets/clientside.mjs

```diff
@@ -0,0 +1,5 @@
+import {display} from "./clientsideModule.mjs";
+
+window.dash_clientside.clientside_module = Object.assign({}, window.dash_clientside.clientside_module, {
```

Comment:
```suggestion
window.dash_clientside = window.dash_clientside || {};
window.dash_clientside.clientside_module = Object.assign({}, window.dash_clientside.clientside_module, {
```
Perhaps my previous comment got lost or wasn't clear - I just don't want to depend on the order in which these scripts execute, to assume that `window.dash_clientside` already exists.

Reply:
Oh, yeah, sorry. Thanks for finding this again!
I was only thinking of not overwriting `window.dash_clientside.clientside_module`, not `window.dash_clientside` itself.